### PR TITLE
Added empty options to component selection

### PR DESCRIPTION
**Pull request by S4mpsa**

### CODE REVIEW

File: hud/powerdisplay.lua [hunk 1]
```diff
@@ -49,6 +49,7 @@ end
 --Only the glass proxy is required, rest have default values.
 function powerDisplay.widget(glasses, data)
     if data ~= nil then
+    if data.state ~= states.MISSING then
```
Comment (gordominossi) [inline hud/powerdisplay.lua:52]: `powerStatus` does not have a `state` property

Comment (S4mpsa) [reply hud/powerdisplay.lua:52]: Then there's something wonky because powerStatus does have a state property.

**Comment by gordominossi** [comment]
You can make branches based on other ones, no need to merge and close PRs so fast

File: hud/powerdisplay.lua (hunk 1 above)
Comment (gordominossi) [reply hud/powerdisplay.lua:52]: Damn

Comment (gordominossi) [reply hud/powerdisplay.lua:52]: True dat, my mistake

**Pull request merged**